runner/src/main.rs

```diff
@@ -305,22 +305,18 @@ fn build_server(opt: &BuildServer) -> Step {
                                 format!("--target={}", opt.server_rust_target.as_deref().unwrap_or(DEFAULT_SERVER_RUST_TARGET)),
                                 "--release".to_string(),
                             ],
-                            ServerVariant::NoIntrospectionClient => vec!["--features=oak_unsafe".to_string(),
+                            ServerVariant::NoIntrospectionClient => vec!["--features=oak-unsafe".to_string(),
```

Comment:
This seems incorrect?

Reply:
I think with `NoIntrospectionClient`, we still want to allow logs (therefore `oak-unsafe` is needed), without having to install npm, which is needed for the introspection client. Without `oak-unsafe`, this server variant will be the same as base.

Reply:
So perhaps this and Base could be merged in the same match arm at some point?

Reply:
`Base` does not log anything, but `NoIntrospectionClient` does. Are you suggesting that we don't need the `NoIntrospectionClient` variant?

Reply:
Ah my bad, I got confused by the various layers of negation :) I think it would be simpler if the name of the variants relates directly to the enabled features, rather than a mix of enabled and disabled features. e.g. unsafe, unsafeWithIntrospection, etc. But also actually, why do we need unsafeWithIntrospection if we already have experimental? I thought the point of the experimental variant is to enable all the features. I guess it may help debugging issues with specific features anyways?

Reply:
That is a good question. Actually I don't know what are the exact use cases for each the variants. I agree with you that we probably only need the following:
- Base
- Experimental (with all `oak_loader` features enabled)
- ExperimentalWithoutIntrospectionClient (similar to Experimental, but without introspection client)
- Coverage (We probably want all the features enabled for it, but it is not currently the case. It is different from Experimental, because in addition to the features it also needs other flags for running coverage)

For debugging specific features it is probably best to also have a way to enable individual features in the `runner` commands, instead of defining new server variants. 

WDYT?